### PR TITLE
Update documentation on ProductSpecifications

### PR DESCRIPTION
**Pull request by victorhmp**
#### What problem is this solving?

Documentation was outdated.

#### Checklist/Reminders

- [x] Updated `README.md`.
- [ ] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
✔️ | Bug fix 
_ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: docs/ProductSpecifications.md [hunk 1]
```diff
@@ -1,68 +1,63 @@
-# Product Specifications
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-## Description
+# Product Specifications
 
 `ProductSpecifications` is a VTEX component that shows the technical specifications of a product.
```
Comment (guerreirobeatriz) [inline docs/ProductSpecifications.md]: `ProductSpecifications` is a block that shows the technical [specifications](https://help.vtex.com/tutorial/what-are-fields-or-specifications--2lB4AgibEseceMggKE2k2m) of a product and its SKUs.

File: docs/ProductSpecifications.md [hunk 2]
```diff
@@ -1,68 +1,63 @@
...
+      "product-specifications#product"
+    ]
+  },
+   "product-specifications#product": {
+    "props": {
+      "shoudCollapseOnTabChange": true,
+      "collapsible": "desktopOnly"
+    }
+  },
 ```
```
Comment (guerreirobeatriz) [inline docs/ProductSpecifications.md]: I would add the following: 

⚠️ The block consumes data from the `product-context` therefore it should only be used in the `store.product` block, that is, in the product template.

File: docs/ProductSpecifications.md [hunk 3]
```diff
@@ -1,68 +1,63 @@
-# Product Specifications
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-## Description
+# Product Specifications
 
 `ProductSpecifications` is a VTEX component that shows the technical specifications of a product.
 This component is meant to be used inside the `store.product` block and consume data from the `product-context`.
```
Comment (guerreirobeatriz) [inline docs/ProductSpecifications.md]: I'd delete this sentence from here and put it in the configuration section. What do you thinj?

File: docs/ProductSpecifications.md [hunk 4]
```diff
@@ -1,68 +1,64 @@
...
+| showSpecificationsTab    | `Boolean`                                                  | Choose if you want to show the component with tabs mode                                                                | `false`       |
+| shoudCollapseOnTabChange | `Boolean`                                                  | If it should collapse if you change the tab                                                                            | `false`       |
+| collapsible              | `mobileOnly`&#124;`desktopOnly`&#124;`always`&#124;`never` | Control when should the content of the specifications be collapsible                                                   | `always`      |
+
+## Customization
+
+In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).
+
+| Class name                                |
```
Comment (Klynger) [inline docs/ProductSpecifications.md]:
```suggestion
| CSS Handles                                |
```

Comment (victorhmp) [reply docs/ProductSpecifications.md]: Eeeeita

**Pull request merged**